modules/freetype/doc/freetype.rst

```diff
@@ -0,0 +1,8 @@
+FreeType Module
```

Comment:
RST-based documentation is not supported anymore. OpenCV uses doxygen to build documentation.